publishable/lang/uk/database.php

```diff
@@ -1,55 +1,32 @@
 <?php
 
 return [
-    'add_bread'                 => 'Додати BREAD до даної таблиці',
     'add_new_column'            => 'Додати новий стовпчик',
     'add_softdeletes'           => 'Додати Soft Deletes',
     'add_timestamps'            => 'Додати мітки часу',
     'already_exists'            => 'Вже існує',
     'already_exists_table'      => 'Таблиця :table вже існує',
     'bread_crud_actions'        => 'BREAD/CRUD дії',
     'bread_info'                => 'BREAD інформація',
+    'browse_bread'              => 'Перегляд',
     'column'                    => 'Стовпчик',
     'composite_warning'         => 'Попередження: цей стовпчик є частиною складового індексу',
-    'controller_name'           => 'Назва контролера',
-    'controller_name_hint'      => 'наприклад, порожній PageController,  буде використовувати BREAD Controller',
-    'create_bread_for_table'    => 'Створити BREAD для таблиці :table',
-    'create_migration'          => 'Створити міграцію для даної таблиці?',
-    'create_model_table'        => 'Створити модель для даної таблиці?',
     'create_new_table'          => 'Створити нову таблицю',
     'create_your_new_table'     => 'Створити вашу нову таблицю',
+    'create_model_table'        => 'Створити модель для даної таблиці?',
     'default'                   => 'За замовчуванням',
-    'delete_bread'              => 'Видалити BREAD',
     'delete_bread_before_table' => 'Перед видаленням таблиці обов\'язково видаліть BREAD таблиці.',
-    'delete_table_bread_conf'   => 'Так, видалити BREAD',
-    'delete_table_bread_quest'  => 'Ви впевнені, що хочете видалити BREAD для таблиці :table?',
     'delete_table_confirm'      => 'Так, видалити таблицю',
     'delete_table_question'     => 'Ви дійсно хочете видалити таблицю :table?',
-    'description'               => 'Опис',
-    'display_name'              => 'Відображена назва',
-    'display_name_plural'       => 'Відображена назва (в множині)',
-    'display_name_singular'     => 'Відображена назва (в однині)',
-    'edit_bread'                => 'Редагувати BREAD',
```

Comment:
have been away from the repo for a while. but why these keys are removed?

Reply:
these keys are removed from english version https://github.com/the-control-group/voyager/blob/1.6/publishable/lang/en/database.php